examples/paths/src/index.ts

```diff
@@ -1,5 +1,3 @@
 import { moduleA } from "@modules/moduleA";
 
 moduleA();
-
-//# sourceMappingURL=index.js.map
```

Comment:
I thought the bug you fixed would let us add linked source map but this PR is only removing them, so I'm confused

Reply:
I'm sorry. I think this is a misunderstanding. The TypeScript file here is a source file. It should not contain a sourceMappingURL comment. Only the generated output should contain a sourceMappingURL comment.

When rules_swc switched to the Rust-based cli, I updated the tests and added the comments manually to the source files (like this TypeScript source file). At that time, it was unclear for me where the sourceMappingURL comment came from and I thought that this was the normal behaviour (no generated sourceMappingURL comment). Personally, I've never used the Node.js @swc/cli before. I'm a direct consumer of the Rust swc_core crate. So I thought, that the sourceMappingURL comment was something that they removed from SWC, because I could not find it in the Rust source code. I did not know that the sourceMappingURL comment was generated in the Node.js @swc/cli. Later, this was reported as a regression bug in https://github.com/aspect-build/rules_swc/issues/146.

This PR restores the original source map tests to the state when rules_swc used the Node.js @swc/cli.

Reply:
I discovered the Rust-based SWC cli around December last year, when the SWC team started to release the cli's together.

The Node.js @swc/cli uses NAPI bindings, but these NAPI binaries are independent from the Rust-based SWC cli. In short, both cli's, the Node.js @swc/cli and the Rust-based SWC cli do the same thing but are totally independent from each other.

Reply:
The upstream bug fix is not complicated and took so long because I wanted to bring the sourceMappingURL generation into the shared swc_core crate to eliminate redundant code and to reduce the software debt. But to my surprise, this is not something that the SWC team is interested in.

Reply:
Oh, haha "this is a typescript file" totally explains it. I didn't realize how we had hacked it to get green tests before.

Yeah I followed your upstream saga with keen interest. Donny is super busy and I don't think he meant to be rude. I'm often the same way. We are way overcommitted and juggling lots of OSS projects with wide scope.

Thanks!